src/utils/helpers.ts

```diff
@@ -54,6 +63,52 @@ const getPlatformInfo = (platformName: string) => {
   else return { shortName: '', icon: {} };
 };
 
+const createDevtoolsDocsUrl = (
+  supportedFramework: string,
+  urlReadySuite: string,
+  suite: string,
+) => {
+  let linkSupported = true;
+
+  let devtoolsDocsURL = `${baseDocsURL}/devtools/tests/${supportedFramework}.html#${urlReadySuite}`;
+  if (
+    suite === removedOldTestDevTools ||
+    nonDocumentedTestsDevTools.includes(suite)
+  ) {
+    devtoolsDocsURL = '';
+    linkSupported = false;
+  }
+  return { devtoolsDocsURL, linkSupported };
+};
+
+const getDocsURL = (suite: string, framework_id: Framework['id']) => {
+  const framework = frameworkMap[framework_id];
+  const supportedFramework =
+    supportedPerfdocsFrameworks[framework as SupportedPerfdocsFramework];
+  const urlReadySuite = suite.replace(/:|\s|\.|_/g, '-').toLowerCase();
+
+  let docsURL = '';
+  let isLinkSupported = true;
+
+  const isDevToolsFramework = framework_id === frameworks[6].id;
```

Comment:
I don't like the fact that if we change the constant then this will no longer be valid. What do you think about defining the devtools framework `{ id: 12; name: 'devtools' }` which will be of type Framework (defined in `/src/types/types.ts`)?

Reply:
Great idea 💯